src/components/DropDetails.js

```diff
@@ -36,7 +42,7 @@ export const DropDetails = ({ title, startOpen, onToggle, children }) => {
             className="drop-arrow-color"
             width="24px"
             height="24px"
-            style={{ transform: `${open ? 'rotate(180deg)' : ''}` }}
+            style={{ transform: `${!open ? 'rotate(180deg)' : ''}` }}
```

Comment:
question: why the inverse behavior?

Reply:
The component wasn’t behaving as expected, so we decided that it made more sense to change the behavior rather than the angle.